src/api.rs

```diff
@@ -13,10 +13,69 @@ pub struct Features {
     pub features: Vec<Feature>,
 }
 
+#[derive(Debug, PartialEq, Clone)]
+pub struct TagFilter {
+    pub name: String,
+    pub value: String,
+}
+
+impl TagFilter {
+    pub fn format(&self) -> String {
+        format!("{}:{}", self.name, self.value)
+    }
+}
+
+#[derive(Serialize, Debug)]
```

Comment:
This might as well derive PartialEq and Clone; while not used in the lib itself, they are awkward for users to add externally